references/cli/dryrun.go

```diff
@@ -45,9 +52,10 @@ import (
 // DryRunCmdOptions contains dry-run cmd options
 type DryRunCmdOptions struct {
 	cmdutil.IOStreams
-	ApplicationFile string
-	DefinitionFile  string
-	OfflineMode     bool
+	ApplicationFiles []string
+	DefinitionFile   string
+	OfflineMode      bool
+	MergeOrphanFiles bool
 }
 
 // NewDryRunCommand creates `dry-run` command
```

Comment:
let's add some explanation about the working mechanism.

1. Add some examples commands.
2. Dry Run with policy and workflow will only take `override/topology` policies and `deploy` workflow step into considerations. Other workflow step will be ignored.